.gitignore

```diff
@@ -6,6 +6,7 @@ build/
 avro_tools/
 avro-fastserde/src/test/java/com/linkedin/avro/fastserde/generated/
 *.class
+avro-fastserde/src/codegen
```

Comment:
why ignoring the codegen?

Reply:
Should not be ignored. I think Sourav hasn't contributed since we introduced this codegen directory as part of the review workflow, so he doesn't know the intent behind it... which reminds me I intended to document it, but haven't yet.

Reply:
Yes, I thought they are jhm generated classes. I will add them back.

Reply:
so we're still waiting on this to be removed... ?